README.md

```diff
@@ -49,7 +53,12 @@ add her to the list of guests in the castle.
 
 Write your code here:
 ```ruby
-# code here
+
+ town[:residents].remove("Belle")
```

Comment:
You could use [delete](http://ruby-doc.org/core-1.9.3/Hash.html#method-i-delete) and store that as a variable